docs/DeployingOnCentOS7.md

```diff
@@ -0,0 +1,291 @@
+     --listen-client-urls "http://0.0.0.0:2379,http://0.0.0.0:4001" \
+     --data-dir /data
+
+# Install and Configure Calico on the Slaves
+
+Calico will need to be running on each of your slaves in order to perform Calico networking. 
+
+To obtain the Calico files, you will need `wget` installed. If you haven't already done so, download the tool with `yum -y install wget`.
+
+## Download the Calico Isolator
+
+    $ wget https://github.com/projectcalico/calico-mesos/releases/download/v0.1.1/calico_mesos
+    $ chmod +x calico-mesos
+    $ sudo mkdir /calico
+    $ sudo mv calico_mesos /calico/calico_isolator
+
+## Create the modules.json Configuration File
+
+    $ cat > modules.json <<EOF
+    {
+      "libraries": [
+        {
+          "file": "/opt/net-modules/libmesos_network_isolator.so",
+          "modules": [
+            {
+              "name": "com_mesosphere_mesos_NetworkIsolator",
+              "parameters": [
+                {
+                  "key": "isolator_command",
+                  "value": "/calico/calico_isolator"
+                },
+                {
+                  "key": "ipam_command",
+                  "value": "/calico/calico_isolator"
+                }
+              ]
+            },
+            {
+              "name": "com_mesosphere_mesos_NetworkHook"
+            }
+          ]
+        }
+      ]
+    }
+    EOF
+    $ sudo mv modules.json /calico/
+
+## Run Calico Node
+
+Download `calicoctl`, Calico's command line tool. We will use `calicoctl` to spin up calico-node in a docker container on each slave. 
+
+When doing so, we must point `calicoctl` to our running instance of etcd. Go back and retrieve the IP address of the host where etcd is running. Use that address in the `ECTD_AUTHORITY` variable passed to `calicoctl`.
+
+    $ wget https://github.com/projectcalico/calico-docker/releases/download/v0.9.0/calicoctl
+    $ chmod +x calicoctl
+    $ sudo ETCD_AUTHORITY=<IP of host with etcd>:4001 ./calicoctl node
+
+# Configure & Run Mesos
+
+An explanation of the configuration options for Mesos can be found by running `mesos-init-wrapper -h`. 
+
+We will be need to set the correct environment variables for each slave and master. These environment variables are interpreted as command line arguments in the mesos-slave and mesos-master applications at runtime.
+
+Once these environment variables are set accordingly, we can finally start the Mesos services.
+
+## Mesos Master
+
+### Set Mesos' Master Environment Variables
+
+Append the following lines to `/etc/default/mesos-master` on each of your master hosts. 
+
+The value of `MESOS_QUORUM` may change depending on how many master hosts you have in your cluster. Mesos recommends that the quorum count is at least 1/2 the number of master hosts running. 
+
+Also note, you will need to fill in the IP address of the host that is running ZooKeeper.
+
+    MESOS_QUORUM=1
+    MESOS_WORK_DIR=/var/lib/mesos
```

Comment:
`MESOS_WORK_DIR` and `MESOS_LOG_DIR` seem like they are vital to any mesos deployment, and thus should be already provided in the RPMs. We should investigate if they're necessary, what the defaults are, and if we should put them in the RPMs, leave them where they are, or remove them altogether. The other two are specific to our demo, so I'm fine with keeping them.